.env.example

```diff
@@ -3,3 +3,5 @@ SSL_ENABLED=false
 DOMAIN=
 EMAIL=
 RENEW_INTERVAL=
+RTMP_AUTH_URL=
+TEST_RTMP_AUTH_KEY= 
```

Comment:
you should just have an "RTMP_AUTH_URL" that you use within the test, not have a separate env var for the test

Reply:
@roddylindsay I have added a commit to fix this. 

The test script will now use "stream1" as the stream key and will get the auth token from the "RTMP_AUTH_TOKEN" environment variable. For the auth URL, it will use the value specified in "RTMP_AUTH_URL", if "RTMP_AUTH_URL" is empty it will default to http://nginx-rtmp:80/auth. 

Let me know if this is what you were looking for.